packages/gatsby/src/redux/reducers/telemetry.ts

```diff
@@ -0,0 +1,42 @@
+import { ActionsUnion, IGatsbyState } from "../types"
+
+const initialState = {
+  gatsbyImageSourceUrls: new Set<string>(),
+}
+
+export const telemetryReducer = (
+  state: IGatsbyState["telemetry"] = initialState,
+  action: ActionsUnion
+): IGatsbyState["telemetry"] => {
+  switch (action.type) {
+    case `PROCESS_GATSBY_IMAGE_SOURCE_URL`: {
```

Comment:
I'm not seeing where this action or the one below would ever get dispatched, is that in a followup PR?

Reply:
oh wait I see the add image action being dispatched, but I don't see the `clearGatsbyImageSourceUrls` action ever used.

Reply:
yup, exactly! :slightly_smiling_face: